package.json

```diff
@@ -51,7 +51,7 @@
     "underscore": "^1.8.3"
   },
   "devDependencies": {
-    "babel": "^5.8.21",
+    "babel": "^5.8",
```

Comment:
if another package requires a lower `5.8` than `5.8.21`, you *should* have two versions, since there's a version conflict. If both packages use `^` then it should auto-dedupe.